crates/sqlexec/src/remote/planner.rs

```diff
@@ -189,6 +190,21 @@ impl ExtensionPlanner for DDLExtensionPlanner {
                     or_replace: lp.or_replace,
                 })))
             }
+            ExtensionType::DescribeTable => {
+                let lp = require_downcast_lp::<DescribeTable>(node);
+                let runtime = if lp.temp || lp.builtin {
+                    RuntimePreference::Local
+                } else {
+                    RuntimePreference::Remote
+                };
```

Comment:
builtin and temp tables can be resolved locally. Native tables need to be resolved on remote.